### PR TITLE
TOC

### LINKED ISSUE #3: It should be possible to link to the particular Quick Start section
Perhaps we should add a table of contents to it as well?

### PR DESCRIPTION
**Pull request by jajakobyly**
Adds table of contents and ability to link to sections in docs (with stable, independent from section's title urls). Finally, I decided to use JS to generate TOC, as this way seems to be the simplest one 

branched from #9 
fixes #3

### CODE REVIEW

File: assets/main.js [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+    };
+
+    var each = wrap(Array.prototype.forEach);
+    var filter = wrap(Array.prototype.filter);
+    var map = wrap(Array.prototype.map);
+    var mapNotUndef = function (obj, fn) {
+        return filter(map(obj, fn), function (it) {
+            return it !== undefined
+        });
+    };
```
Comment (jajakobyly) [inline assets/main.js]: JavaScript :heart: (sarcasm)

Comment (matklad) [inline assets/main.js]: Not sure what happens here, but perhaps we can use `$.each`, `$.grep`, e.t.c here?

File: assets/main.js [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+
+        installBtn.addEventListener("click", function (e) {
+            installPane.classList.toggle("visible");
+            e.target.blur();
+            e.preventDefault();
+        });
+
+        installPaneCloseBtn.addEventListener("click", function (e) {
+            installPane.classList.remove("visible"
+            );
```
Comment (matklad) [inline assets/main.js]: Something happend with formatting here

**Comment by matklad** [comment]
Why you are not using jQuery selectors here? I find them more concise then default DOM APIs. We already have jQuery, though I am not sure why...

File: assets/main.js (hunk 1 above)
Comment (jajakobyly) [inline assets/main.js]: We don't have such things here

File: assets/main.js (hunk 2 above)
Comment (jajakobyly) [inline assets/main.js]: Uh missed this. Seems like formatter bug in IntelliJ

**Comment by jajakobyly** [comment]
We don't have jury here, and I don't want to use it as it's pretty heavy

**Comment by matklad** [comment]
Ah, sorry, I was assuming we were using jQuerry already, but turns out that Chrome Dev Tools just always define `$` symbol in console[](url)

**Comment by jajakobyly** [comment]
It's currently selected node in DOM inspector afaik

**Pull request merged**